src/components/structures/SpaceHierarchy.tsx

```diff
@@ -226,6 +230,8 @@ const Tile: React.FC<ITileProps> = ({
                 }}
             >
                 { description }
+                { topic && " · " }
+                { topic && topic }
```

Comment:
As Sonar is saying, the `&&` isn't necessary, something `&&`-ed with itself will always equal itself

Reply:
My bad. I assumed it would render `undefined` without the check but looks like it doesn't. Fixed.